### DISCUSSION TITLE
Urgent: Compromised GitHub Account - Username: deepak16080

### DISCUSSION THREAD

**Discussion by deepakyadav-developer** [community/community#133353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I am writing to inform you that my GitHub account has been compromised. Below are the details of my account and the issue I am facing:

GitHub Username: deepak16080
Email Associated with the Account: dy16080@gmail.com

Description of the Issue:

Account Compromise:

I suspect that my account has been hacked as I am unable to sign in using my usual credentials.
I have tried the password recovery options, but I am still unable to regain access to my account.
Unauthorized Activity:

I have noticed unauthorized activities on my account, including changes to repositories and commits that I did not authorize.
Steps Taken:

I have changed the password for my associated email account to ensure its security.
I have reviewed my email for any suspicious activities or security alerts from GitHub.
Request:

Please assist me in securing my account and regaining access to it.
Kindly investigate any unauthorized changes and revert them if possible.
If additional verification is required, please let me know the steps I need to follow.
I am deeply concerned about the security of my projects and data stored in my GitHub account. I would appreciate your prompt assistance in this matter.

Thank you for your support.

Best regards,
Deepak Yadav

**Comment by ebndev** [comment]
Hi @deepak160080,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
